pkg/provider/azure_storageaccount.go

```diff
@@ -72,6 +72,9 @@ type AccountOptions struct {
 	SubnetName                              string
 	AccessTier                              string
 	MatchTags                               bool
+	EnableBlobVersioning                    bool
```

Comment:
it's better using `*bool`, then it would have 3 values: true, false, nil, otherwise you cannot distinguish false and nil condition